### PR TITLE
added data preprocessing tool

### PR DESCRIPTION
**Pull request by likileads**
This is a data preprocessing template which would help begineer like me to:
-  import all the basic essential libraries for data processing
-  importing dataset
- know about dependent and independent variables
- taking care of missing data in the dataset
- encoding categorical data
- splitting the dataset into the training set and test set
- feature scaling

### CODE REVIEW

**Review by jlooper — COMMENTED** [review]
Hi, thank you but this contribution does not align to our lesson design or curriculum scope

**Comment by likileads** [comment]
Hi @jlooper, where can I find the lesson contents page.
Please help me out. As a beginner, I wish to learn by contributing.

**Comment by jlooper** [comment]
We’re not looking for contributions to lessons as they are complete. We welcome human-generated translations and typo fixes. Check the discussion boards for learning opportunities with other community members.
